README.md

```diff
@@ -156,4 +156,4 @@ namespace FileUploader
 
 ## Explore Further
 * [Complete Documentation](https://docs.minio.io)
-* [Minio .NET SDK API Reference](https://docs.minio.io/docs/.NET-client-api-reference)
\ No newline at end of file
+* [Minio .NET SDK API Reference](https://docs.minio.io/docs/.NET-client-api-reference)
```

Comment:
actually this was @deekoder's suggestion. dotnet- prefix definitely sounds better. will change it